main/src/library/Path.flix

```diff
@@ -175,6 +211,22 @@ namespace Path {
                 writeAll(xs, w)
         }
 
+    ///
+    /// Writes an array of bytes to the specified file
+    ///
+    pub def writeBuffer(buffer: Array[Int8], p: Path): Result[Unit, IOError] & Impure = try {
+        import new java.io.FileOutputStream(##java.io.File) as newFileOutputStream;
+        import java.io.FileOutputStream.write(Array[Int8]);
+
+        let Path(javaFile) = p;
+        let outputStream = newFileOutputStream(javaFile);
+        outputStream.write(buffer);
+        Ok(())
+    } catch {
+        case _: ##java.lang.SecurityException => Err(Generic("Security Error"))
```

Comment:
I think we should just let all security exceptions flow through.

Reply:
Yes, agreed. I missed this one when I cleaned the other occurrences. Thanks for the heads up!